tests/test_core/test_alignment.py

```diff
@@ -2479,15 +2479,14 @@ class AlignmentTests(AlignmentBaseTests, TestCase):
 
     def test_sliced_deepcopy(self):
         """correctly deep copy aligned objects in an alignment"""
-        path = "data/brca1_5.paml"
-        aln = load_aligned_seqs(path, array_align=False, moltype="dna")
-        aln = self.Class(aln.named_seqs)
-        aln = aln[20:30]
-        for name in ["Mouse", "Human", "HowlerMon", "NineBande", "DogFaced"]:
-            new_seq = aln.named_seqs[name].deepcopy()
-            self.assertEqual(len(new_seq.data), 10)
-            new_seq = aln.named_seqs[name].deepcopy(sliced=False)
-            self.assertEqual(len(new_seq.data), len(aln.named_seqs[name].data))
+        data = {"seq1": "ACGACGACG", "seq2": "ACGACGACG"}
+        orig = self.Class(data)
+        aln = orig[2:5]
+        for name in orig.names:
+            copied = aln.named_seqs[name].deepcopy()
+            self.assertNotEqual(len(copied.data), len(aln.named_seqs[name].data))
```

Comment:
change to `self.assertLessThan()`
change copied to `sliced.named_seqs[name].data`, `notsliced.named_seqs[name].data`.